backend/src/main/java/com/example/dontwastetomuch/game/GameService.java

```diff
@@ -92,15 +97,15 @@ public void removeMyGame(MyUser myUser, String gameId){
             myUser.getGameData().remove(data);
             myUserRepo.save(myUser);
         } else {
-            throw new IllegalArgumentException("Game not found");
+            throw new IllegalArgumentException("Game not found!");
         }
 
 
     }
 
     public List<GameData> getAllMyGames(MyUser myUser) {
-        if (myUser.getGameData().isEmpty()){
-            throw new NoSuchElementException("Add some games to your list");
+        if (myUser.getGameData() == null || myUser.getGameData().isEmpty()){
```

Comment:
Maybe already create GameData on user Object? That way it will never be null and could only be empty

Reply:
thought about it, but for me it was the easy way. Don't know what is the right way :D